Create Introduction “our-calls” (#117)
sourcecred/docs: docs/intro/our-calls.md

---
title: Our Calls
description: A list of they types of calls, meetings, work sessions, and jams we
  have in our internal community.
---
In SourceCred we frequently work asynchronously, but we also make use of voice and/or video meetings when we need to get coordinated. There are a few different types of calls we have in our community, below you'll find a list of our regular calls and their expectations. 

All calls/meetings take place in the voice channels of the SourceCred Discord server.

> You can see what types of calls are currently scheduled by looking at our \[calendar](http://sourcecred.io/calendar).

Reviewer: ```suggestion
You can see what types of calls are currently scheduled by looking at our [calendar](http://sourcecred.io/calendar).
```

# Weekly Community Call

> Audience: Anyone who is interested in SourceCred, especially newcomers

Reviewer: ```suggestion
Audience: Anyone who is interested in SourceCred, especially newcomers.
```

> What: The founder, and some members of the core team host a call for any newcomers or community members to attend, introduce themselves, ask questions, etc.

Reviewer: Can we just make it "the core team"? That way if I miss some meetings, we're not defaulting on this promise.

Why: To provide newcomers with the opportunity to get to know the project and its main team, or ask any questions they have about SourceCred. 

> Where: Discord Main voice channel

Reviewer: ```suggestion
Where: Discord Main voice channel.
```

> When: Weekly, Wednesdays 11am Pacific Time

Reviewer: ```suggestion
When: Weekly, Tuesday at 11am Pacific Time. Check out the [SourceCred Calendar](https://sourcecred.io/calendar).
```

# Jams

What: a designated time to talk about a specific subject and how it relates to SourceCred.

Audience: Anyone who finds that particular topic interesting or relevant to their work in SourceCred.

> Why: Creates a low-barrier way to gather minds who want to think about a specific topic and how it can be applied to our SourceCred goals near or long term. It's fun. To produce interesting thoughts and perspectives on the given topic. 

Reviewer: ```suggestion
Why: We come together to explore a specific topic, and how we can apply it to SourceCred. They are a fun and low-barrier way to get involved in the project.
```

> Where: Discord Main voice channel

Reviewer: ```suggestion
Where: Discord Main voice channel.
```

> When: No set time or reoccurrence. Jams are scheduled as one-off events, timing is usually based off the availability of key participants.

Reviewer: ```suggestion
When: No set time or reoccurrence. Jams are scheduled as one-off events, timing is usually based off the availability of key participants. Check out the [SourceCred Calendar](https://sourcecred.io/calendar).
```

# Weekly Contributor Call

What: A weekly call which focuses on keeping contributors on the same page about larger goals, getting feedback on projects/initiatives, and unblocking contributors where needed.

Audience: This call is meant for people who are actively contributing to SourceCred and need/want to be tuned into the larger picture. If you are about to start contributing, it is definitely okay for you to come listen in as a way to gain context but be mindful of derailing the conversation with basic context questions. If you find yourself lost at the Contributor Call, attend a Community Call instead and ask questions to your heart's content. Being a contributor does not make this meeting mandatory for you. 

Why: This call gives all of the contributors the opportunity to see the bigger picture and align their actions/priorities with the overall goals of the project. It also gives the branch leads a chance to get feedback and assistance on larger goals/changes. Helps grow a sense of connection and shared goals across different portions of SourceCred.

> Where: Discord Main voice channel

Reviewer: ```suggestion
Where: Discord Main voice channel.
```

> When: Weekly, Thursday 11am PST

Reviewer: ```suggestion
When: Weekly, Thursday at 11am PST.
```

# Branch Specific Meetings

What: Meetings for contributors working in a specific branch to coordinate, check in on progress, and align goals and action with each other and the branch lead. 

Audience: The contributors who are working on initiatives within that specific branch, and the branch lead. 

Why: Keeps teams effective, aligned, and communicating. Helps grow a sense of connection and shared goals within the branch. Gives contributors a chance to receive help, propose new ideas, and communicate changes.

> Where: Discord Main voice channel

Reviewer: ```suggestion
Where: Discord Main voice channel.
```

> When: At a regular day/time of the team's choosing.

Reviewer: ```suggestion
When: At a regular day/time of the team's choosing. Find them on the [SourceCred Calendar](https://sourcecred.io/calendar).
```

# Work Sessions

What: A generally shorter and faster paced session scheduled for a specific team to come together and make progress on a specific goal or initiative. Having shorter sessions with an emphasis on getting a few singular goals completed can make these sessions more productive than regular meetings. 

Audience: The contributors on a team that is actively working on a specific initiative and any other high-context contributors they need feedback from. Anyone who plans to start contributing to the specific initiative in question can attend to gain context. 

Why: Sometimes you just need some synchronous space with all the key players to really make progress on an initiative or project. By nature of being shorter, these are also less taxing and can help keep contributors on track.

> Where: Discord Main voice channel

Reviewer: ```suggestion
Where: Discord Main voice channel.
```
